### PR TITLE
Obfuscate sensitive vals in console

### PR DESCRIPTION
**Pull request by pselle**
Updates `terraform console` to show `(sensitive)` when a value is marked as sensitive.

### CODE REVIEW

File: repl/format.go [hunk 1]
```diff
@@ -37,6 +37,9 @@ func FormatValue(v cty.Value, indent int) string {
 			return fmt.Sprintf("null /* %s */", ty.FriendlyName())
 		}
 	}
+	if v.IsMarked() {
+		return "(sensitive)"
+	}
```
Comment (alisdair) [inline repl/format.go:42]: This probably doesn't practically matter, but I'd expect a sensitive variable with a `null` value to still display `(sensitive)` in the console, if only because I think it would do so in a plan. I think checking for the mark here instead of before the `if v.IsNull()` branch means it displays `null` instead.

**Review by alisdair — APPROVED** [review]
Good catch & nice test coverage! I have one edge case comment below but I don't think it's important.

File: repl/format.go (hunk 1 above)
Comment (pselle) [reply repl/format.go:42]: Excellent point! And easy enough to move that ordering.

**Comment by pselle** [comment]
Props go to @mildwonkey for finding the panic/bug! Thanks for the review!
